src/main/java/frc/robot/util/RobotMath.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.util;
+
+public class RobotMath {
+    public static double powerCurve(double x) {
+        // https://www.desmos.com/calculator/g07ukjj7bl
+        double curve = (0.5D * (Math.atan(Math.PI * (Math.abs(x) - 0.5D)))) + 0.5D;
+        return Math.copySign(curve, x);
+    }
+
+    public static void normalize(double[] array) {

```

Comment:
This function needs documentation badly. As is, I have no idea what it does (OK, after reading it a few times I get a picture, but still.)

Also, an array parameter named `array` is... less than ideal. This falls under the same umbrella as documenting the method to begin with.

Reply:
Also, is this ever useful outside of `Drive`? (another thing documentation would help.) If not... see comments on moving of `powerCurve`.